src/domexpression.js

```diff
@@ -49,7 +60,16 @@ var _ = Mavo.DOMExpression = $.Class({
 		if (typeof this.expression !== "string") { // Still unhandled?
 			if (this.attribute) {
 				// Some web components (e.g. AFrame) hijack getAttribute()
-				var value = Element.prototype.getAttribute.call(this.node, this.attribute);
+				const originalGetAttribute = Element.prototype.getAttribute;
```

Comment:
Since this never changes, you can move it outside the class altogether. (Not a blocker for merging this PR)

Reply:
Done! Thanks.